### ISSUE TITLE
tray icon does not change without a restart

### ISSUE THREAD

**Issue by dark-penguin** [FreeRDP/Remmina#981]
When you enable "Dark tray icon" in Preferences, the icon does not change until you restart Remmina. There is no mention of this in the option description. It would probably be nice to either reload the icon immediately, or mention "(requires restart)" near that option.

**Comment by antenore** [comment]
Good point! I doubt we can reload the icon, if it's not possible we will add the description.

Thanks!

**Comment by ic3d** [comment]
Hi @dark-penguin we're reviewing old open issues. Do you still get this one? Thanks!

**Comment by box293** [comment]
When I was testing the color_schemes branch recently this was still a problem, needed to restart for it to take effect.

**Comment by antenore** [comment]
Gosh... I forgot about this :-\

I'll take it.

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @box293 @dark-penguin @ic3d
